cobrand_hackney/static/app.scss

```diff
@@ -236,3 +236,29 @@ ul.case-list-user li {
         padding: 0 0 0 5px;
     }
 }
+
+// Merging menu
+.lbh-merging-bar {
+    background-color: #00664f47;
+    padding: 1em 0;
+    margin-top:-2.25em;
+    // margin-right: -1em;
+    // margin-left: -1em;
+    border: 1px solid #00664e5d;
+    border-top: none;
+
+    form {
+        margin-top:1em;
+    }
+
+    @media (min-width: 40.0625em) {
```

Comment:
We should probably be using one of the pre-defined govuk/lbh-frontend breakpoints here, eg:

```scss
@include govuk-media-query($from: tablet) {
```

These are the default breakpoints defined in lbh-frontend: https://github.com/LBHackney-IT/lbh-frontend/blob/18ab4ec49593d57cc45fd4d19423907178b261f2/lbh/vendor/_sass-mq.scss#L39-L44

Reply:
Thank you for that when I looked on the source code I could not find the rules.